core/trino-main/src/main/java/io/trino/execution/DynamicFilterConfig.java

```diff
@@ -332,4 +344,12 @@ public DynamicFilterConfig setLargeMaxSizePerFilter(DataSize largeMaxSizePerFilt
         this.largeMaxSizePerFilter = largeMaxSizePerFilter;
         return this;
     }
+
+    public void applyFaultTolerantExecutionDefaults()
+    {
+        smallPartitionedMaxDistinctValuesPerDriver = 100_000;
+        smallPartitionedMaxSizePerDriver = DataSize.of(100, KILOBYTE);
```

Comment:
Why not increase `*RangeRowLimitPerDriver` as well ?
That limit could be kept at 2x the distinct values limit.